src/lib/configure.js

```diff
@@ -27,7 +28,7 @@ module.exports = create => config => {
   // https://github.com/sindresorhus/ky/pull/153
```

Comment:
do we still need this ? its been merged

Reply:
This is not the PR I thought it was, it still gets `fetch` off the global object so we can't pass a custom fetch implementation. This comment should just be removed.